server/graphql/v2/object/OAuthAuthorization.js

```diff
@@ -38,6 +39,19 @@ export const OAuthAuthorization = new GraphQLObjectType({
       type: new GraphQLNonNull(GraphQLDateTime),
       description: 'The time of expiration',
     },
+    lastUsedAt: {
+      type: GraphQLDateTime,
+      description: 'The last time of token was used',
+      async resolve(authorization) {
+        const activity = await models.Activity.findOne({
+          where: {
```

Comment:
Small perf improvements: add a `SELECT` on the query to get only the field we're interested in

```suggestion
          attributes: ['createdAt'],
          where: {
```